cmd/kubeadm/app/cmd/init.go

```diff
@@ -22,6 +22,7 @@ import (
 	"html/template"
 	"io"
 	"io/ioutil"
+	"os"
```

Comment:
With testing I notice that in this file we're outputting:

[preflight] Starting the kubelet systemd service by running "systemctl start kubelet"
Using Kubernetes version: v1.4.6

Where that is about the only line in all output that doesn't have a [something] prefix.